dashboard/checkit/dashboard.py

```diff
@@ -91,9 +92,9 @@ def build(*args):
         o = outcomes_dropdown.value
         output.clear_output()
         with output:
-            display(Markdown("Generating 1,000 seeds..."))
+            display(Markdown("Generating %s seeds..."%amount_box.value))
```

Comment:
I think `f"Generating {amount_box.value} seeds..."` is a bit more readable here.

Reply:
I am learning many things about string formatting, lol